doc/sphinx/addendum/generalized-rewriting.rst

```diff
@@ -886,7 +886,7 @@ succeeds, making progress using a reflexivity proof of rewriting.
 ``progress`` tests progress of the argument :n:`@rewstrategy` and fails if no
 progress was made, while ``try`` always succeeds, catching failures.
 ``choice`` is left-biased: it will launch the first strategy and fall back
```

Comment:
```suggestion
``choice`` uses the first successful strategy in the list of @rewstrategy.
```
and drop "on the next one in case of failure" on the next line.